src/PipelinesMicroserviceCLI/Commands/CLICommand.php

```diff
@@ -0,0 +1,27 @@
+<?php
+namespace PipelinesMicroserviceCLI\Commands;
+
+use Symfony\Component\Console\Command\Command;
+use GuzzleHttp\Client;
+
+abstract class CLICommand extends Command
```

Comment:
Ik zou dit misschien hernoemen naar PipelineManagerAPICommand om aan te geven wat de specialisatie van deze base klasse is.

Of je kan die specialisatie ook naar een Trait verplaatsen